fraud/service_test.go

```diff
@@ -352,7 +348,8 @@ func TestService_GossipingOfBEFP(t *testing.T) {
 	require.NoError(t, err)
 
 	newCtx, cancel := context.WithTimeout(ctx, time.Millisecond*100)
-	t.Cleanup(cancel)
+	defer cancel()
```

Comment:
Nit but let's be consistent with way we cancel context. I see some places use t.Cleanup and some defer